client/src/utils/users.js

```diff
@@ -31,4 +31,53 @@ export const getUserProfile = () => {
       "x-access-token": localStorage.getItem('token')
     }
   })
+}
+
+export const getUserProfileById = (userId) => {
+  return axios.get('http://localhost:8888/users/profile-id', {
```

Comment:
You shouldn't commit localhost based paths into main. These should probably be relative. Something like

```javascript
axios.get('/users/profile-id', {
```